From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 3 Feb 2020 14:46:15 +0100
Subject: [PATCH 03/26] info-overview: Use the Ubuntu logo prominently

This removes GNOME branding in favour of Ubuntu branding. We don't
make changes to the text information as the distribution/OS is clearly
visible above the GNOME version.

See https://bugzilla.gnome.org/show_bug.cgi?id=695691

(Patch taken from Fedora and updated for Ubuntu)
---
 panels/info-overview/cc-info-overview-panel.ui | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/panels/info-overview/cc-info-overview-panel.ui b/panels/info-overview/cc-info-overview-panel.ui
index de377bb..5333246 100644
--- a/panels/info-overview/cc-info-overview-panel.ui
+++ b/panels/info-overview/cc-info-overview-panel.ui
@@ -30,7 +30,8 @@
                   <object class="GtkImage">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="resource">/org/gnome/control-center/info-overview/GnomeLogoVerticalMedium.svg</property>
+                    <property name="pixel_size">128</property>
+                    <property name="icon_name">ubuntu-logo-icon</property>
                   </object>
                   <packing>
                     <property name="expand">False</property>
